.github/workflows/ci.yaml

```diff
@@ -40,7 +40,11 @@ jobs:
         cabal:
           - "3.6.2.0"
         os:
-          - ubuntu-latest
+          # Using Ubuntu 20.04 due to recent GLIBC_2.34 version mismatch errors for
```

Comment:
Ok great! So if I compiled with older version of glibc, it is likely to work with the new version? If that is so that is great! We could then always ensure that we use a bit later ubuntu than the newest one, so that we don't get again into this situation where people have too old glibc.

Reply:
Correct, everything I have found online seems to state the following:

> One of the GNU C Library's (glibc's) unwritten rules is that a program built against an old version of [glibc](https://www.gnu.org/software/libc/) will continue to work against newer versions of glibc.
- https://developers.redhat.com/blog/2019/08/01/how-the-gnu-c-library-handles-backward-compatibility

> Glibc has a versioning system that allows backward compatibility (older programs built to run on older versions of glibc will continue to run on new glibc); but it is of no help the other way around: programs that depend on newer glibc will usually not run on systems with older glibc.
- https://www.lightofdawn.org/wiki/wiki.cgi/NewAppsOnOldGlibc

And again, this is how it used to be working for us before GitHub made the change to `ubuntu-latest`. Up until Nov, it was already actually compiling everything against 20.04 :) and it worked for people on latest Linux versions to the best of our knowledge (i.e. you and Filip never had problems). So I think we are safe to be 1 behind latest for compiling.

Reply:
Awesome! Let's explain in the comment that we are using a bit older Ubuntu version to ensure that glibc is not too knew, since stuff compiled on older glibc is compatible with new glibc but not other way around. So in the future, if we are tempted to update the ubuntu version, we will know to what to update it.